### DISCUSSION TITLE
first person to reply will be marked as answer | 🧠 Galaxy brain 🧠 achievement (please help me aswell)

### DISCUSSION THREAD

**Discussion by 1RB** [Schweinepriester/github-profile-achievements#480] [Q&A]
im not sure what im supposed to do but um help.

**Comment by 1RB** [comment]
test

**Reply by muhammadnurulahsan** [reply]
How do I get the Galaxy Brain badge?

**Comment by muhammadnurulahsan** [comment]
How do I get the Galaxy Brain badge?

**Reply by 1RB** [reply]
answer discussions, go to discussions made by other people and ask them to mark your msg as the answer

**Comment by kovsu** [comment]


**Reply by 1RB** [reply]
gotchu

**Reply by kovsu** [reply]
> gotchu

thx :)

**Comment by Preetiraj3697** [comment]
Hello

**Comment by Chameleon-Coder** [comment]
me please

**Comment by LauraBigoni** [comment]
For galaxy brain you need to suggest answers to questions posted in discussion. If your answer is helpful then the person who asked the question can mark you answer as the solution. Once you've had 2 questions marked as solutions then you'll get galaxy brain.

Here's the link to the general GitHub community discussions. It might be a good place to start. https://github.com/community

If you found this useful then please mark my answer as a solution so I can get the badge.

**Comment by Chameleon-Coder** [comment]
answer goes here
